pynhm/hydrology/PRMSCanopy.py

```diff
@@ -161,10 +175,10 @@ def calculate(self, time_length, vectorized=False):
     def calculate_procedural(self, time_length):
 
         # todo: verify that hru_ppt is prcp
-        hru_ppt = prcp
-        potet = potet
-        hru_rain = rainfall
-        hru_snow = snowfall
+        hru_ppt = self.hru_ppt
```

Comment:
@langevin-usgs you should verify that I got these all correct (esp given the large atol) i was just hacking to make it work.